product_pricelist_margins/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Product Pricelist Margins",
+    "summary": "This module shows the product's cost and margin from the pricelists.",
+    "version": "16.0.1.0.0",
+    "category": "Product",
+    "website": "https://github.com/OCA/product-attribute",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": ["product"],
```

Comment:
```suggestion
    "depends": ["account"],
```
``account_tax.compute_all()`` function is in account module

Reply:
This one is still pending.